app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -257,6 +270,41 @@ class Core(private val context: Context) {
         }
     }
 
+    private fun getCookiePolicy(): CookiePolicy {
+        return if (context.settings().blockCookiesInCustomTrackingProtection) {
+            when (context.settings().blockCookiesSelectionInCustomTrackingProtection) {
+                "all" -> CookiePolicy.ACCEPT_NONE
+                "social" -> CookiePolicy.ACCEPT_NON_TRACKERS
+                "unvisited" -> CookiePolicy.ACCEPT_VISITED
+                "third-party" -> CookiePolicy.ACCEPT_ONLY_FIRST_PARTY
+                else -> CookiePolicy.ACCEPT_NONE
+            }
+        } else {
+            CookiePolicy.ACCEPT_ALL
+        }
+    }
+
+    private fun getTrackingCategories(): Array<TrackingCategory> {
+        val categories = arrayListOf(
+            TrackingCategory.AD,
+            TrackingCategory.ANALYTICS,
+            TrackingCategory.SOCIAL,
+            TrackingCategory.CONTENT,
+            TrackingCategory.MOZILLA_SOCIAL,
+            TrackingCategory.SCRIPTS_AND_SUB_RESOURCES
+        )
```

Comment:
I'm waiting to validate which categories we should apply here
![image](https://user-images.githubusercontent.com/773158/74451705-09612b00-4e4e-11ea-848b-2c15ccca5983.png)

Reply:
By default we just need to add [`AD`,`ANALYTICS`,`SOCIAL`,`MOZILLA_SOCIAL`]

- If the user check/uncheck **tracking content**, we need to append/remove `SCRIPTS_AND_SUB_RESOURCES`
- If the user check/uncheck **Cryptominers**, we need to append/remove `CRYPTOMINING` 
- If the user check/uncheck **Fingerprinters**, we need to append/remove `FINGERPRINTING`

Reply:
> If the user check/uncheck tracking content, we need to append/remove SCRIPTS_AND_SUB_RESOURCES

And `CONTENT` ?

Reply:
It's not need. It applies a more restricted list that we are not actually using on desktop neither on Fenix right now.